### PR TITLE
refactor: new-structure

### PR DESCRIPTION
**Pull request by vojtech-cerveny**
New structure of Page object pattern - start. It should be more clear what is on the page and page has some assertions. 

Let me know if I should continue or we are OK with old structure. :)

### CODE REVIEW

File: pages/new-structure/components/navigation-bar.ts [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+
+  readonly userIsLogged: Locator;
+  readonly profileButton: Locator;
+  readonly profileDropdownItem: Locator;
+  readonly logoutDropdownItem: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+
+    const nav = page.locator("nav");
```
Comment (PanLuce) [inline pages/new-structure/components/navigation-bar.ts]: more self-explanatory variable name could be present here

File: pages/new-structure/components/navigation-bar.ts [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+   * Clicks on the home link.
+   */
+  async clickHome() {
+    await this.home.click();
+  }
+
+  /**
+   * Clicks on the "For Parents" link.
+   */
+  async clickForParents() {
```
Comment (PanLuce) [inline pages/new-structure/components/navigation-bar.ts:80]: I don't view this as low-code variant. It has been debated on various meetups that these "shortcut" methods are only more code and no better readability. But I feel we can teach this as we want.

File: pages/new-structure/components/navigation-bar.ts [hunk 3]
```diff
@@ -0,0 +1,227 @@
+import { expect, type Locator, type Page } from "@playwright/test";
```
Comment (PanLuce) [inline pages/new-structure/components/navigation-bar.ts:1]: don't forget to change the folder structure to the actual one prior to merging

**Review by PanLuce — APPROVED** [review]

**Comment by PanLuce** [comment]
> New structure of Page object pattern - start. It should be more clear what is on the page and page has some assertions.
> 
> Let me know if I should continue or we are OK with old structure. :)

Ok. Just don't forget to change the folder structure to the actual one prior to merging.

File: pages/new-structure/components/navigation-bar.ts (hunk 2 above)
Comment (vojtech-cerveny) [reply pages/new-structure/components/navigation-bar.ts:80]: Hmm... yeah, we can teach both. With and without.

**Pull request merged**